### ISSUE TITLE
Request random UA function add browser selection

### ISSUE THREAD

**Issue by viper645** [PurityWasHere/Anomaly-Mod-Hosting#2]
Can u pls add in function random user-agent that we can choose what browser it will generate like the main browsers in that function are chrome, firefox and opera

**Comment by viper645** [comment]
> As of right now there is no random user-agent function, however you can set a custom user-agent for the bot using the user-agent header. (Devs, correct me if I'm wrong.)

No there is in function block you can use random UA but u can't choose what browser you want to get.
Now its random UA between like Chrome, Firefox and Opera for sure maybe there are more

**Comment by nonrect** [comment]
> 
> 
> > As of right now there is no random user-agent function, however you can set a custom user-agent for the bot using the user-agent header. (Devs, correct me if I'm wrong.)
> 
> No there is in function block you can use random UA but u can't choose what browser you want to get.
> Now its random UA between like Chrome, Firefox and Opera for sure maybe there are more

True, But it would've been great if we could choose the browser and also platform (mobile or desktop) if possible
